toolchains/cpu.spack.gyselalibxx_env/linux-gyselalibxx-env-0.23.0.yaml

```diff
@@ -0,0 +1,43 @@
+spack:
+  concretizer:
+    unify: true
+  definitions:
+  - compilers:
+    - 'gcc@11:'
+  - packages:
+    - 'cmake@3.30'
+    - 'eigen@3.4'
+    - 'fftw@3.3 precision=float,double'
+    - 'ginkgo@1.8'
+    - 'googletest@1.14 +gmock'
+    - 'openmpi@5.0'
+    - 'paraconf@1.0'
+    - 'pdi@1.9 -fortran -python'
+    - 'pdiplugin-decl-hdf5@1.9 +mpi -fortran'
+    - 'pdiplugin-decl-netcdf@1.9 +mpi'
+    - 'pdiplugin-mpi@1.9'
+    - 'pdiplugin-set-value@1.9'
+    - 'pdiplugin-trace@1.9'
```

Comment:
I thought `-` was not much advertised but reading the doc it seems that both are valid and both have problems, see https://spack.readthedocs.io/en/latest/basic_usage.html#variants. I would just advertise to use only one notation for coherency.

Reply:
[05346a5](https://github.com/gyselax/gyselalibxx/pull/260/commits/05346a55c0e14985d1f456b3094101187ba31bc7)